spec/actions/get_user_from_omniauth_action_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GetUserFromOmniauthAction, type: :action do
+  describe '#perform_implementation' do
+    context 'when the user exists' do
+      let!(:user) { create(:user) }
+      let(:existing_user_token) { create(:access_token, info: { name: user.name, email: user.email }) }
```

Comment:
```suggestion
      let(:existing_user_token) { create(:access_token, info: {name: user.name, email: user.email}) }
```

(it's interesting why Rubocop has no complains about that)

Reply:
It's by default: [SpaceInsideHashLiteralBraces](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceInsideHashLiteralBraces)